instruqt-tracks/terraform-cloud-aws-v2/02-terraform-cloud-setup/check-workstation

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -l
+set -e
+
+# Create /tmp/skip-check to disable this check
+if [ -f /tmp/skip-check ]; then
+    rm /tmp/skip-check
+    exit 0
+fi
+
+export ORG=$(grep organization /root/terraform-cloud/terraform.tfvars | cut -d '"' -f2)
+export WORKSPACE=$(grep workspace /root/terraform-cloud/terraform.tfvars | cut -d '"' -f2)
+
+agent variable set TF_ORG $ORG
+agent variable set TF_WORKSPACE $WORKSPACE
+
+cp /root/hashicat-aws/remote_backend.tf.example /root/hashicat-aws/remote_backend.tf
```

Comment:
Lines 16-24 we seem to be doing setup for future challenges. 
1. They rely only on agent variables we have set, doesn't it make sense to do this in the setup scripts for the appropriate challenges—these aren't checking the current challenge
2. Should be be hiding that we're doing work from the student?

Reply:
Learners do not get access to Instruqt variables. These are accessible via assignment (MD) files, or challenge lifecycle scripts.

Reply:
Ah, okay, I missed that distinction.